sys/arduino/include/SPI.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPI_H
```

Comment:
When I was reviewing the documentation, I realized that this file needs some header with copyright notice, license and documentation as any other file. To be consistent, the documentation should be similar to tht in `Arduino.h` and `Wire.hpp`.

Reply:
I added the standard boiler plate to it :-)

Reply:
Thanks. We should decide in which group `Arduino.h`, `Wire.h` and `SPI.h` are. All of these wrappers should be in the same group. At the moment `Arduino.h` and `Wire.h` are in group `sys_arduino` while `SPI.h` is `sys_arduino_api`. It would be ok for me to place all of them in group `sys_arduino_api`. What do you think?

Reply:
I'm fine with either. But having this consistent would be much better. Maybe for now move `SPI.h` to `sys_arduino` to keep this PR focused on SPI? If someone prefers `sys_arduino_api` over that, it could be moved together in a trivial PR.